adapters/cadent_aperture_mx/cadent_aperture_mxtest/supplemental/bad-imp-video-missing-w.json

```diff
@@ -0,0 +1,145 @@
+{
+  "mockBidRequest": {
+    "id": "some_test_auction",
+    "imp": [
+      {
+        "id": "some_test_ad_id",
+        "video": {
+          "mimes": [
+            "video/mp4",
+            "application/javascript"
+          ],
+          "protocols": [
+            2,
+            3,
+            5,
+            6
+          ],
+          "h": 480
```

Comment:
Cadent: What should the behavior be if just one size dimension is missing? Previous behavior would set the other one to 0.